idl/chromadb/proto/coordinator.proto

```diff
@@ -384,4 +392,5 @@ service SysDB {
   rpc FlushCollectionCompaction(FlushCollectionCompactionRequest) returns (FlushCollectionCompactionResponse) {}
   rpc RestoreCollection(RestoreCollectionRequest) returns (RestoreCollectionResponse) {}
   rpc ListCollectionVersions(ListCollectionVersionsRequest) returns (ListCollectionVersionsResponse) {}
+  rpc GetCollectionSize(GetCollectionSizeRequest) returns (GetCollectionSizeResponse) {}
```

Comment:
what do you think about calling this

```suggestion
  rpc GetApproximateCollectionSize(GetCollectionSizeRequest) returns (GetCollectionSizeResponse) {}
```

since this is reading from the replica and it may lag behind the leader?